src/Site/views/languageforge/container/languageforge.html.twig

```diff
@@ -151,28 +151,6 @@
     {% endif %}
 {% endblock %}
 
-{% block footer %}
-    <footer class="footer">
-        <div class="d-flex">
-            <div class="d-none d-sm-flex align-items-end">
-                <div>
-                    <a href="https://inter.payap.ac.th/"><img src="/Site/views/shared/image/payap_logo.png"
-                                                          alt="Payap University Logo" width="85" height="25" /></a>
-                    <a href="http://www.sil.org"><img src="/Site/views/shared/image/sil_logo_small.png"
```

Comment:
do we need to get that logo in the app somewhere else? If not, I'd suggest removing it from the repo if it's not being used elsewhere.

Reply:
The logo and the other text should still be a part of the brochure home page footer.  Didn't check the links but they should be the same

Reply:
ok, thanks